tbx/blog/tests/test_models.py

```diff
@@ -15,45 +14,8 @@ def test_create(self):
 class TestBlogPageFactory(WagtailPageTestCase):
     def test_create(self):
         blog_post = BlogPageFactory()
-        self.assertEqual(blog_post.related_services.count(), 1)
+        self.assertEqual(blog_post.related_services.count(), 0)
 
         services = ServiceFactory.create_batch(size=3)
         another_blog_post = BlogPageFactory(related_services=list(services))
         self.assertEqual(another_blog_post.related_services.count(), 3)
```

Comment:
Also check `related_sectors`

```suggestion
        services = ServiceFactory.create_batch(size=3)
        sectors = SectorFactory.create_batch(size=2)
        another_blog_post = BlogPageFactory(related_services=list(services), related_sectors=list(sectors))
        self.assertEqual(another_blog_post.related_services.count(), 3)
        self.assertEqual(another_blog_post.related_sectors.count(), 2) 
```